lib/Cake/Test/Case/View/Helper/FormHelperTest.php

```diff
@@ -8793,7 +8802,7 @@ public function testCreateQuerystringrequest() {
 		));
 		$expected = array(
 			'form' => array(
-				'id' => 'ContactAddForm',
+				'id' => 'ContactActionForm',
```

Comment:
This is a side effect of always using the action of the `url` array, before the action was incorrectly defaulted to `add` whereas the action clearly states `'action' => 'action'`.
So this was corrected in the process to respect the actual action name if given.